charts/hivemq-edge/templates/statefulset.yml

```
@@ -0,0 +1,187 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "hivemq-edge.name" (dict "releaseName" .Release.Name) }}
+  labels:
+      {{- include "hivemq-edge.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "hivemq-edge.selector-labels" . | nindent 6 }}
+  serviceName: "hivemq-edge"
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "hivemq-edge.selector-labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: hivemq-edge
+          image: "{{ .Values.image.repository }}:{{.Values.image.tag}}"
+          imagePullPolicy: Never
```

This needs to be dynamic, otherwise, won't work with this image pull policy.
We should use the already available value `image.imagePullPolicy`

```suggestion
          imagePullPolicy: {{ .Values.image.pullPolicy }}
```